### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by oleksis** [BlackPythonDevs/blackpythondevs#12] [Ideas]
Logo / Branding 

We can improve the Promp together

Ex: Using DALL•E 3

- [1](https://www.bing.com/images/create/create-a-logo-with-this-features3a---have-as-refere/64e20f5543044479bf4917a438081eae?id=cyDuZxwUhWMReLW4H7jyLA%3d%3d&view=detailv2&idpp=genimg&FORM=GCRIDP&mode=overlay)
- [2](https://www.bing.com/images/create/create-a-logo-with-this-features3a-0a--have-as-refer/652a781cfec84637a4e5f206b4dd5e2b?id=N6WYwrxKOb41s9BGY%2BcyEQ%3D%3D&view=detailv2&idpp=genimg&form=GCRIDP)

**Comment by kjaymiller** [comment]
This conversation should move to .maintainers
